builders/manage/sudo.md

```diff
@@ -0,0 +1,105 @@
+
+!!! note
+    This tutorial demonstrates assigning arbitrary token balances on a TestNet ContainerChain that has no value. You should carefully consider the ramifications of creating additional tokens on your own ContainerChain.
+
+### Checking Existing Account Balance  {: #checking-existing-account-balance }   
+
+The next section will demonstrate how to assign arbitrary token balances to accounts using the Sudo account. This process will overwrite the specified account's existing balance, so verifying the account is empty is a good practice before continuing. To check an account's balance, take the following steps:
+
+1. Navigate to the **Developer** Tab of Polkadot.js Apps and click on **Chain State**
+2. Select the **system** pallet to query
+3. Select **account** 
+4. Paste in the account address or select it from the dropdown
+5. Press **+** icon
+6. You'll see the balance information returned at the bottom, including free, reserved, and frozen balances
+
+![Check balances on Polkadot.js Apps](/images/builders/manage/sudo/sudo-5.webp)
+
+### Assigning Balances with Sudo  {: #assigning-balances-with-sudo }  
+
+To assign an account balance to an account, make sure to have your Sudo account accessible in Polkadot.js Apps. Then, take the following steps: 
+
+1. Navigate to the **Developer** Tab of Polkadot.js Apps for your ContainerChain
+2. Click on **Sudo**. If you do not see **Sudo** in this menu, then you have not associated the Sudo account with Polkadot.js Apps. Make sure that your Sudo account is injected by your wallet and connected to Polkadot.js Apps
+3. Select the **balances** pallet
+4. Select the **forceSetBalance** method
+5. Paste in the account address to endow with tokens or select it from the dropdown
+6. Enter the amount of tokens to endow the account with. In this example, we specify `9000000000000000000` for nine native tokens. Remember that EVM ContainerChains have 18 decimals while Substrate or Custom ContainerChains decimals are configured when launching the chain. If you're unsure how many decimals your ContainerChain has, navigate to the **Settings** tab and Click on **Metadata**
+7. Press **Submit Sudo** and confirm the transaction in your wallet
+
+![Force assign balances on Polkadot.js Apps](/images/builders/manage/sudo/sudo-6.webp)
+
+## Changing the Sudo Account {: #changing-the-sudo-account }
+
+Changing your ContainerChain's Sudo account is a straightforward process. Also known as rotating your sudo keys, this process will remove sudo access from the existing sudo account and grant it to the new account. There can only be one sudo account at any time. However, you are free to change the sudo account as often as you would like.
+
+Prior to getting started, make sure that you have your existing Sudo account accessible in Polkadot.js Apps. Then, take the following steps:
```

Comment:
If a user skips ahead to this section and, for whatever reason, they aren't familiar with Polkadot.js Apps, will they know what to do? I don't think we should really link to Polkadot.js Apps, but maybe have a very brief section or add it to the prereqs that they should have Polkadot.js Apps connected to their ContainerChain (with instructions on how to do so) and then in these other sections where we say Polkadot.js Apps it can be a link back to the section with instructions, just in case haha 😅